projects/SSTorytime/services/sstorytime/module.nix

```diff
@@ -0,0 +1,168 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  inherit (lib)
+    mkEnableOption
+    mkPackageOption
+    mkOption
+    mkIf
+    types
+    ;
+
+  cfg = config.services.sstorytime;
+
+  # only create a local database if we're gonna connect to it locally
+  localDB = (cfg.database.host == "localhost") || (cfg.database.host == "/run/postgresql");
```

Comment:
@eljamm , one last comment. Yesterday I came across multiple nixpkgs modules using `database.createLocally`. It looks like this might be some convention now. Would you please consider to use it again. Thank you for your patience.